### PR TITLE
Main branch rename

### PR DESCRIPTION
**Pull request by mo-c4t**

### CODE REVIEW

**Comment by mo-c4t** [comment]
replace foundation with network domain
deleting old workflows for creating dockers
renaming main branch to c4t

**Review by havan — APPROVED** [review]
Looks good to me

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -1,82 +0,0 @@
-name: build-app
```
Comment (c4t-ag) [inline .github/workflows/build.yaml:1]: IMO a good source for ideas - I'm against removal

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -1,49 +0,0 @@
-name: docker
```
Comment (c4t-ag) [inline .github/workflows/docker.yml:1]: Why is this one removed?

**Review by c4t-ag — CHANGES_REQUESTED** [review]

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**